.circleci/config.yml

```diff
@@ -84,6 +85,13 @@ executors:
           POSTGRES_DB: postgres
 
 commands:
+  prepare_go_mod:
+    description: Sets project to act as go module
+    steps:
+      - run:
+          name: Go mod
+          command: go mod init
```

Comment:
Shouldn't we try to migrate the whole repository to go modules instead of this?

Reply:
I'd think it would be preferable, but upstream currently isn't set up for this and there could likely be some fallout in the process.